### ISSUE TITLE
there is no option for preference in ubuntu 22 (jammy)

### ISSUE THREAD

**Issue by dhanwanit** [iptux-src/iptux#502]
I installed iptux with a different method but still in ubuntu 22 there is no option for preference.

**Comment by lidaobing** [comment]
Hi, I did not get the meaning of this issue. Could you explain it more clear with a screenshot? thanks.

**Comment by YJPrakash** [comment]
Yes, I am having the same issue. Kali Linux shows the chat preference menu in the chat conversational dialog. But it never shows up in the ubuntu Iptux version.

**Comment by YJPrakash** [comment]
Ubuntu Xorg session is able to show a preference menu under the sub-chat dialog. Please change it before you enter a password. Choose a session from the listed options like Ubuntu and Ubuntu Xorg. 

![Screenshot from 2023-01-21 12-29-56](https://user-images.githubusercontent.com/35760608/213847998-fceb2d3c-2d37-4199-8458-c4d98950c2b4.png)

Here I am able to see the Application menu in the menu bar.

**Comment by YJPrakash** [comment]
Here mentioned issue only reflects on the Ubuntu Gnome desktop Environment only Kali, and Debian's Desktop Environment works fine.

**Comment by lidaobing** [comment]
fixed in v0.8.4
